lib/tufts/curation/file_set_indexer.rb

```diff
@@ -33,29 +29,6 @@ def generate_solr_document
           solr_doc['duration_tesim']          = object.duration
           solr_doc['sample_rate_tesim']       = object.sample_rate
           solr_doc['original_checksum_tesim'] = object.original_checksum
-
-          # ead customizations
-          if object.mime_type == "text/html"
-            results = {}
-
-            find_indexable_fields(results)
-            # puts "#{results}"
-            results.each do |field_name, field_values|
-              # puts("  #{field_name}")
-              if solr_doc["#{field_name}_tesim"].nil?
-                solr_doc["#{field_name}_tesim"] = field_values
-              else
-                solr_doc["#{field_name}_tesim"] += field_values
-              end
-            end
-            # Only do this after the indexer has the file_set
-            load_ead_xml(object)
-            if @noko.nil?
-              Rails.logger.warn("Couldn't find the Ead XML for #{solr_doc['id']}")
-            else
-              solr_doc['all_text_timv'] = @noko.xpath('//text()').text.gsub(/[^0-9A-Za-z]/, ' ')
-            end
-          end # end if
```

Comment:
I'm not sure if this is right, I realize it says #ead customizations but, do we just want to remove lines 52-57 or this whole block?

Reply:
I think you're right - although I'm not totally sure how that method worked for non-EAD items.  I put it back minus the load_ead_xml(object) call and if @noko.nil? block.